### PR TITLE
Support Multiple Dir CERT-1730

### PR DESCRIPTION
**Pull request by NoaBablikiCertora**
**Support multiple repos in one workspace!!**

- alternate between one repo / multiple repo work
- compatible to the filesystem in the workspace

### CODE REVIEW

**Comment by NoaBablikiCertora** [comment]
===KINDA DONE===
needs to check for bugs manually and then code review

**Comment by NoaBablikiCertora** [comment]
Pretty much done
needs to update the CHANGELOG and version number (package.json) to 0.2.0

File: src/extension/extension.ts [hunk 1]
```diff
@@ -18,10 +18,15 @@ import {
   Job,
 } from './types'
 import { checkDir } from './utils/checkDir'
-import { Run } from '../results/types'
-import { getProgressUrl } from './utils/getProgressUrl'
+// import { Run } from '../results/types'
```
Comment (Biced) [inline src/extension/extension.ts:21]: delete comments

File: src/extension/ScriptRunner.ts [hunk 2]
```diff
@@ -392,6 +386,65 @@ export class ScriptRunner {
...
+            return rs
+          })
+        }
+      }
+      return vrLink
+    }
+  }
+
+  private async checkFileValidity(fileUri?: Uri): Promise<string> {
+    try {
```
[the remark this answers is not in the document]
Comment (NoaBablikiCertora) [reply src/extension/ScriptRunner.ts:436]: yes because it doesn't exists

**Pull request merged**